src/components/organisms/AdminVenueView/components/EventsView/EventsView.tsx

```diff
@@ -21,23 +24,33 @@ import { Checkbox } from "components/atoms/Checkbox";
 import "./EventsView.scss";
 
 export type EventsViewProps = {
-  venueId: string;
-  venue: WithId<AnyVenue>;
+  venueId?: string;
+  venue?: WithId<AnyVenue>;
+  worldId?: string;
+  variant: EventsVariant;
 };
 
-export const EventsView: React.FC<EventsViewProps> = ({ venueId, venue }) => {
+export const EventsView: React.FC<EventsViewProps> = ({
+  variant,
+  venueId,
+  venue,
+  worldId,
+}) => {
   const {
-    findVenueInRelatedVenues,
     relatedVenueIds,
     isLoading: isVenuesLoading,
+    findVenueInRelatedVenues,
   } = useRelatedVenues({
     currentVenueId: venueId,
   });
+
   const { events, isLoaded: isEventsLoaded } = useSpaceEvents({
-    worldId: venue.worldId,
+    worldId: venue?.worldId ?? worldId,
     spaceIds: relatedVenueIds,
   });
 
+  const isWorldEvent = variant === EventsVariant.world;
```

Comment:
Looks like now I can see all the world experiences in all the spaces. And current space and world schedules differ only in the Checkbox "Split by Space", although we can see all the experiences in the space schedule as well.

shouldn't we filter and show the Experiences that are related to the specific space if it's the Schedule tab in the space settings? 
and if it's a world schedule, then we show all the experiences.

Reply:
> shouldn't we filter and show the Experiences that are related to the specific space if it's the Schedule tab in the space settings?
> and if it's a world schedule, then we show all the experiences.

That is exactly what we are trying to achieve, yes